lib/bindings/python/tests/test_aic_capacity.py

```
@@ -241,6 +242,22 @@ def test_pad_nextn_accept_rates_rejects_invalid(bad):
+    assert _nextn_accepted_from_accept_rates(1, "0.85,0.3") == pytest.approx(0.85)
+    assert _nextn_accepted_from_accept_rates(2, "0.85,0.3") == pytest.approx(
+        0.85 + 0.85 * 0.3
+    )
+    assert _nextn_accepted_from_accept_rates(3, "1,0.5,0.25") == pytest.approx(
+        1.0 + 0.5 + 0.125
+    )
+
+
+@pytest.mark.parametrize("nextn", [0, 6])
+def test_nextn_accepted_rejects_unsupported_mocker_depth(nextn):
+    with pytest.raises(ValueError, match="nextn must be 1..=5"):
```

Ruff (0.15.21)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)